sdk/analytics/src/test/java/com/klaviyo/analytics/linking/DeepLinkingTest.kt

```diff
@@ -0,0 +1,170 @@
+package com.klaviyo.analytics.linking
+
+import android.app.Activity
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.net.Uri
+import android.os.Bundle
+import com.klaviyo.core.Registry
+import com.klaviyo.fixtures.BaseTest
+import com.klaviyo.fixtures.MockIntent
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.unmockkAll
+import io.mockk.verify
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+
+internal class DeepLinkingTest : BaseTest() {
```

Comment:
similar to above, renamed file and changed enough that it became a delete/add in the diff.